server/src/lib/model/user-client.ts

```diff
@@ -280,10 +280,9 @@ const UserClient = {
     );
     // the accountClientId can't be a placeholder value otherwise it'll
     // treat any ? in the username or email as a placeholder also and the query will break
-
-    updateDemographics(accountClientId, data.age, data.gender);
-
+    const updateDemographicsPromise = updateDemographics(accountClientId, data.age, data.gender);
     await Promise.all([
+      updateDemographicsPromise,
```

Comment:
Not doing it inline the array because I want the attached comment to stay in-context.